config/routes.rb

```diff
@@ -4,7 +4,7 @@
   devise_for :users
   root to: 'movies#index'
   resources 'aws_texts' , only:[:index, :show]
-  resources 'questions'
+  resources 'questions', only: [:index]
```

Comment:
質問の新規投稿機能がありますので，少なくとも`create` は必要になります。

また，元のアプリでは質問の一覧表示と新規投稿は両方 `index` にあるのですが，現在 `new` の方に新規投稿ページがあるようです。

元のアプリに合わせるのであれば `new` を消去して中身を `index` の方に移動させ，ヘッダーのリンクも `index` にとぶように修正する必要があります。

```suggestion
  resources 'questions', only: [:index, :create]
```

もし，一覧表示と新規投稿を分けるのであれば，ルーティングに `new` を追加し，一覧表示に新規投稿ページへのリンクをつけるなどの対応が必要でしょう。